### PR TITLE
active_enrolment

### PR DESCRIPTION
**Pull request by jdfox**
# Motivation and Context
As a part of the refactor of action service, update case table adding `active_enrolment` to replace `actionplanid` in the future

# What has changed
- added `active_enrolment` to case table and case service OpenAPI doc
- removed abandonware cobertura plugin, incompatible with Java 8

# How to test?

# Links
- https://trello.com/c/ShvG3lxu

### CODE REVIEW

File: src/main/resources/database/changes/release-10.37.0/casesvc_bres_foundation_schema.sql [hunk 1]
```diff
@@ -48,6 +48,7 @@ CREATE TABLE "case" (
     collectioninstrumentid character varying(128),
     state character varying(20),
     actionplanid integer,
+    active_enrolment boolean,
```
Comment (warrenbailey) [inline src/main/resources/database/changes/release-10.37.0/casesvc_bres_foundation_schema.sql:51]: You can't do this.

The database is managed by liquibase, so you will need to do an alter table statement in a new release

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: src/main/resources/database/changes/release-10.37.0/casesvc_bres_foundation_schema.sql (hunk 1 above)
Comment (jdfox) [reply src/main/resources/database/changes/release-10.37.0/casesvc_bres_foundation_schema.sql]: Fixed.

File: src/main/resources/database/changes/release-22/changelog.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+
+- changeSet:
+    id: 22-1
+    author: Jon Fox
+    changes:
+    - sqlFile:
+        comment: Add active_enrolment column to case table
+        path: add-activeenrolment-to-case.sql
```
Comment (warrenbailey) [inline src/main/resources/database/changes/release-22/changelog.yml:9]: nit picking here - but could we have a dash between active and enrolment for readabilty

File: openapi.yaml [hunk 3]
```diff
@@ -602,6 +602,8 @@ components:
       properties:
         actionPlanId:
           type: string
+        active_enrolment:
```
[the remark this answers is not in the document]
Comment (warrenbailey) [reply openapi.yaml:605]: Though I can't see where you've changed the DTO to reflect this api change?

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jdfox) [reply openapi.yaml:605]: Removing for now per offline discussion: too soon for this to be in the API.

File: src/main/resources/database/changes/release-22/changelog.yml (hunk 2 above)
Comment (jdfox) [reply src/main/resources/database/changes/release-22/changelog.yml:9]: Done.

File: src/main/resources/database/changes/release-22/add-activeenrolment-to-case.sql [hunk 4]
```diff
@@ -0,0 +1,2 @@
+alter table casesvc."case"
+add column activeEnrolment boolean;
```
Comment (warrenbailey) [inline src/main/resources/database/changes/release-22/add-activeenrolment-to-case.sql:2]: This shouldn't be camelCase, all other columns in the table are lower case. Please change to "activeenrolment" for consistency.

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: src/main/resources/database/changes/release-22/add-activeenrolment-to-case.sql (hunk 4 above)
Comment (warrenbailey) [reply src/main/resources/database/changes/release-22/add-activeenrolment-to-case.sql:2]: Also I thought we were going to put a dash in the file name between active and enrolment?

Comment (jdfox) [reply src/main/resources/database/changes/release-22/add-activeenrolment-to-case.sql]: done.

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**